docs/reference/run.md

```diff
@@ -590,7 +590,10 @@ the container exits**, you can add the `--rm` flag:
 
 > **Note**: When you set the `--rm` flag, Docker also removes the volumes
 associated with the container when the container is removed. This is similar
-to running `docker rm -v my-container`.
+to running `docker rm -v my-container`. Only volumes that are specified without a
+name are removed. This includes volumes inherited via `--volumes-from`. For example, with
```

Comment:
I'm wondering something on that sentence. From reading it, I was wondering if doing a `docker run --rm --volumes-from=data-container…` would remove or not the unamed volumes from the container `data-container` — by reading it my first guess was that it would remove them, which it does not.

```
$ docker volume create --name=one
$ docker volume create --name=two
$ docker run -dti -v one:/one -v /three busybox /bin/sh
$ docker run --rm -it --volumes-from silly_hugle -v /four -v two:/two busybox /bin/sh
# While running
DRIVER              VOLUME NAME
local               one
local               two
local               b01400488037b97d25dcd0b1049f3f0c9c1c8f462843ba1f2b1c2fe92d75ca71
local               7a98cabdc8acbb3d1f7790041bc8c0798febc8ae98c61d3fce428d1b57d97f36
# After leaving the `--rm` container
DRIVER              VOLUME NAME
local               one
local               two
local               b01400488037b97d25dcd0b1049f3f0c9c1c8f462843ba1f2b1c2fe92d75ca71
```

I think we could be a little clearer on this, @cpuguy83 wdyt ?